Automattic/jetpack
# My jetpack connection footer logos to list

Fixes https://github.com/Automattic/jetpack/issues/35486

## Proposed changes:

Use list for the items on Jetpack Connection page footer to address accessibility issues

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a new JN site using this branch
* Navigate to My Jeetpack page and click 'Finish setting up Jetpack'
* Check that icons on the left side in footer look normal and are the part of the list.
